plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4143,7 +4158,9 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
         // @mechanism LocationTag.sign_glow_color
         // @group world
         // @description
+        // Pre 1.20
         // Returns the name of the glow-color of the sign at the location.
+        // Optionally provide a side (defaults to "front")
```

Comment:
Same here, probably a separate tag to get the back's glow color, that way it can properly have a matching mechanism as well